### PR TITLE
feat: added Satcom 3D Model

### PR DESCRIPTION
**Pull request by Henrik2k17**
## Summary of Changes

This Pull Request adds the ability to toggle the satcom on the fuselage.

-Added Honeywell Jetwave 3D Model including custom made textures
-Added a Simvar
-Added option to toggle the satcom in the EFB ( Aircraft Options) 

## Screenshots (if necessary)
FlightSimulator_vIykSxC0bb
i_view64_FNpI2s2rOB

## References
d-sites-devnewmodstoreaero_1025010134-data-uploadedimages-modification-0e2dafbb-74c0-4457-b48f-605f183af1cd-2
brave_n9wY9y1KxT

## Additional context
Discord username (if different from GitHub): Henrik#0777

## Testing instructions
Spawn with any livery and see if the satcom is displayed correctly without any issues or bugs.

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

**Review by bouveng — APPROVED** [review]

**Comment by NUTTTTTTTT** [comment]
Quality Assurance Tester Report

Discord: █▀█ █▄█ ▀█▀#2123
Object of testing: (#PR 7470)
Tier of Testing: 1
Date: (04/09/2022)

Testing Process: 
- Loaded in C+D , tested 4 liveries.
- Enabled/Disabled SATCOM in EFB: showed/vanished as expected.
- Checked model: Looks great!
- Checked persistence: satcom on/off state persists.

Negatives:       
When up extremely close there's a slight gap between the Asobo fuselage and SATCOM, however not visible unless you try really hard to get right up super close, not visible unless you're inches from the actual SATCOM and won't be visible in general use. 

Given the model was made to match spec/measurements exactly, this is likely more of an Asobo model problem than the satcom, so this would likely fit better on a custom model.

Testing Results: 
Passed

Note: The SATCOM EFB toggle key will need to be added to Localazy for this to show correctly in a production build, I've brought this up with @frankkopp who's going to add this. :)

Conclusions: 
Great addition! Really good first PR. 🚀 
A side note potentially for future PRs, this could be added as a livery specific option in future, similar to the fuel persistence so airframes could be saved with/without SATCOM enabled. 

Media:       
image
image
image
image
image

**Comment by frankkopp** [comment]
Localazy key added. Next build will have the label.

File: src/behavior/src/A32NX_Exterior.xml [hunk 1]
```diff
@@ -225,4 +225,15 @@
...
+    <Template Name="FBW_Satcom">
+    <DefaultTemplateParameters>
+        <SAT_ENABLE></SAT_ENABLE>
+    </DefaultTemplateParameters>
+    <Component ID="#NODE_ID#" Node="#NODE_ID#">
+        <UseTemplate Name="ASOBO_GT_Visibility_Code">
+            <VISIBILITY_CODE>#SAT_ENABLE#</VISIBILITY_CODE>
+        </UseTemplate>
+    </Component>
+</Template>
```
Comment (Saschl) [inline src/behavior/src/A32NX_Exterior.xml:237]: I think the intendation is not quite correct here ;)

**Comment by Henrik2k17** [comment]
The gap between the satdome and fuselage is now fixed 👍

**Comment by mico975** [comment]
Quality Assurance Tester Report

Discord          : mico#3145
Object of testing: #7470
Tier of Testing  : 1
Date             : 05/09/2022

Testing Process: 
Spawn with any livery and see if the satcom is displayed correctly without any issues or bugs. - **OK**

Negatives:       
N/A

Testing Results: 
Passed

Conclusions: 
LGTM

**Comment by Henrik2k17** [comment]
Accidentally closed this PR

**Comment by sidnov** [comment]
> Accidentally closed this PR

Reopened 👍🏻

**Comment by 2hwk** [comment]
IMO the SATCOM model could be on by default instead of off, may be worth trying out this change.

**Comment by Henrik2k17** [comment]
I did think about it, although not many airframes have a satcom. If it's wished I can change it. 

> IMO the SATCOM model could be on by default instead of off, may be worth trying out this change.

**Comment by 2hwk** [comment]
> I did think about it, although not many airframes have a satcom. If it's wished I can change it.
> 
> > IMO the SATCOM model could be on by default instead of off, may be worth trying out this change.

On second thoughts, this would be something we should have on a per livery basis eventually, which is out of the scope of this PR.

**Pull request merged**